README.md

```diff
@@ -101,7 +101,13 @@ You love FastHub? You want new features or bug fixes?
 Please **contribute** to the  project either by [_creating a PR_](https://github.com/k0shk0sh/FastHub/compare) or [_submitting an issue_](https://github.com/k0shk0sh/FastHub/issues/new) on GitHub.  
 Read the [**contribution guide**](.github/CONTRIBUTING.md) for more detailed information.
 
-- [Translation contributors list](.github/language_contributors.md)
+### [Language Contributors](.github/language_contributors.md)
+
+- Turkish [@kutsan](https://github.com/kutsan)
+- German [@failex234](https://github.com/failex234)
+- Chinese [@Devifish](https://github.com/Devifish)
+- Indonesian [@dikiaap](https://github.com/dikiaap)
+- Russian [@dedepete](https://github.com/dedepete)
 
 # License
```

Comment:
Hello replying to review comment from phone

_Sent from my Htc m8 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_